Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/storage/FamilyTestDatabase.kt

```diff
@@ -80,6 +80,12 @@ interface FamilyCoronaTestDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     suspend fun insert(entity: FamilyCoronaTestEntity)
 
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
We should consider (for the future) to not use insert with replace and write an upsert method. See this post for some details:  https://tech.bakkenbaeck.com/post/room-insert-update
I know we are not using foreign key constraint at the moment, but who knows, this could change in the future. (optional)